LectureNotes/Lectures/Fall2018/Lecture8/ASPLecture8.tex

```diff
@@ -0,0 +1,237 @@
+
+
+% ==============================================================================
+%
+\begin{document}
+%
+% ==============================================================================
+
+% \makelecture %my computer says this is undefined...%
+\textbf{\Large October 26 and 29}
+
+\section{A Field Guide to Generators}
+
+\begin{tabular}{|c|p{6cm}|c|}
+\hline
+\textbf{Operator applied to $f$}
+	& \textbf{Description}
+	& \textbf{Example} \\	
+\hline
+	$\frac{d}{dx}f(x)$
+	&  Deterministic movement with speed $+1$
+	&  $X_t = X_0 + t$\\
+\hline
+	$f(y) - f(x)$
+	&  Jump from $x$ to $y$ at rate $1$
+	& $X_t=\begin{cases}x, & \text{if } t<\tau \\ y, & \text{if } t\geq\tau \end{cases}$,
+		where $ \tau\sim Exp(1)$\\	
+\hline
+	$\frac{1}{2}\frac{d^2}{dx^2}f(x)$
+	&  Standard Browninan motion
+	& $B_t$ \\	
+\hline
+	$aGf(x)$
+	& The process corresponding to $G$, run at speed $a$
+	& $X_t = Y_{at}$, where $Y$ has generator $G$\\
+\hline
+	$G_1 f(x) + G_2 f(x)$
+	& Both $G_1$ and $G_2$ ``happen independently"
+	& $\frac{d}{dt} + \frac{1}{2} \frac{d^2}{dx^2} \longrightarrow B_t + t$\\
+\hline
+	$\begin{aligned} &(G_1 \otimes I + I \otimes G_2) f(x, y) \\ &= G_1 f(x,y) + G_2 f(x, y) \end{aligned}$
+	& Independent (multidimensional) processes
+	& $X_t = (Y_t^{(1)} , Y_t^{(2)})$ \\
+\hline
+	$a(x) G f(x)$, $a(x)>0$
+	& The process corresponding to $G$ run at speed $a(x)$ when in state $x$.
+	& $-x \frac{d}{dx}$ on $(0,\infty) \longrightarrow X_t = X_o e^{-t}$ \\
+\hline
+	$\vec{a}(x) \cdot \vec{\nabla} f(x)$, $x\in\R^n$
+	& Deterministic flow along $\vec{a}(x)$
+	& --- \\
+\hline
+	$\begin{aligned} \frac{1}{2}\Delta f(x) = \frac{1}{2}\sum\limits_{k=1}^{n}\frac{d^2}{dx_k^2} f(x),\\ x\in\R^n \end{aligned}$
+	& $n$-dimensional Brownian motion
+	& $B_t = (B_t^{(1)}, \dots, B_t^{(n)})$, i.i.d. BMs $B_t^{(k)}$ \\
+\hline
+	$\sum\limits_{i,j} \alpha_{ij}\left( f(x_j) - f(x_i)  \right)$
+	& Jump to $x_j$ from $x_i$ at rate $\alpha_{ij}$
+	& --- \\
+\hline
+	$\int \alpha(x, y) \left( f(y) - f(x)  \right) dy$
+	& Jump to $y$ from $x$ at rate $\alpha(x, y)dy$
+	& --- \\
+\hline
+\end{tabular}
+
+\section{Examples of Generators}
+\begin{itemize}
+\item \underline{Recall from before}: we defined $X_t^{(M)} = \frac{N_{Mt} - Mt}{\sqrt{M}}$, which converges to Brownian motion as $M\rightarrow\infty$.
+
+In other words, it drifts down at speed $\sqrt{M}$ with jumps of magnitude $\frac{1}{\sqrt{M}}$ at rate $M$.
+
+So, the generator is:
+$$\begin{aligned}
+-\sqrt{M} \frac{d}{dx} f(x) + M\left( f(x + \frac{1}{\sqrt{M}}) - f(x) \right)
+\end{aligned}$$
+
+\item \underline{Resetting}: Suppose a process drifts up at speed $1$ and jumps back to $0$ at rate $1$.
+
+The generator is then:
+$$\begin{aligned}
+\frac{d}{dx} f(x) + \left( f(0) - f(x) \right)
+\end{aligned}$$
+
+\item \underline{Bathtub}: Suppose a bath is filling at a speed of $1$ gal/min, but a kid jumps in at independent intervals given by $exp(1)$ and splashes out a random, uniformly chosen proportion of water.
```

Comment:
Instead of "intervals given by $exp(1)$" how about "at independent random intervals with Exponential(1) distribution"